website/frontend/templates/docs/options.html

```diff
@@ -29,6 +29,34 @@ <h1 id="general">General</h1>
           want Picard to do this scan automatically. In any case, you can direct Picard to scan a particular
           music file at any time using the <i>Scan</i> button in the toolbar.
         </li>
+        <li>
+          <b>Check for updates during start-up:</b> Check this box if you want Picard to check for program updates
+          automatically. Leave it unchecked if you don't want Picard to do this check automatically. In any case, you
+          can have Picard check for program updates at any time using the <i>Help</i> --&gt; <i>Check for update</i>
+          command in the drop-down menu.
+        </li>
+        <li>
+          <b>Days between checks:</b> This option allows you to limit the automatic update checking by selecting
+          the interval, in days, between checks. Set this to <i>1</i> if you want to check daily, <i>7</i> for weekly
+          checks, and so on. Not ethat this only applies if the <i>Check for updates during start-up</i> option is
```

Comment:
Small typo: "Not ethat" -> "Note that"

Reply:
VERY good catch!  Change made.  :slightly_smiling_face: